### PR TITLE
Adds Crystallize Mana to Magiclysm

### PR DESCRIPTION
**Pull request by Kenshkrix**
SUMMARY: Mods "Adds Crystallize Mana spell and scroll to Magiclysm."

Adds Spell: Crystallize Mana
Adds Effect: Mana Fatigue
Adds Item: Scroll of Crystallize Mana

### CODE REVIEW

File: data/mods/Magiclysm/Spells/classless.json [hunk 1]
```diff
@@ -1,4 +1,41 @@
...
+  {
+    "id": "crystallize_mana",
+    "type": "SPELL",
+    "name": "Crystallize Mana",
+    "description": "Crystallizes mana into solid form",
+    "valid_targets": [ "self" ],
+    "spell_class": "NONE",
+    "flags": [ "PERMANENT", "NO_LEGS", "CONCENTRATE" ],
+    "difficulty": 3,
+    "max_level": 1,
```
Comment (KorGgenT) [inline data/mods/Magiclysm/Spells/classless.json]:
```suggestion
```

File: data/mods/Magiclysm/Spells/classless.json [hunk 2]
```diff
@@ -1,4 +1,41 @@
...
+    "min_damage": 1,
+    "max_damage": 1,
+    "min_duration": 1,
```
Comment (KorGgenT) [inline data/mods/Magiclysm/Spells/classless.json]:
```suggestion
```

File: data/mods/Magiclysm/Spells/classless.json [hunk 3]
```diff
@@ -1,4 +1,41 @@
...
+    "max_duration": 2,
```
Comment (KorGgenT) [inline data/mods/Magiclysm/Spells/classless.json]:
```suggestion
```

**Comment by Kenshkrix** [comment]
Since the spell permanently creates a valuable item from nothingness, the idea is that it's a costly ritual which takes a great deal of time and effort.
Effectively you can spend an entire day creating only a few bits of crystallized mana, but they each allow you to cast a high level combat spell if you have the proper wand.
The actual numbers were arrived at during a conversation on Discord with Aptronym.

**Pull request merged**